apps/passport-server/src/database/queries/telegram/insertTelegramConversation.ts

```diff
@@ -31,8 +31,7 @@ export async function insertTelegramChat(
     `\
     insert into telegram_chats(telegram_chat_id)
     values ($1)
-    on conflict (telegram_chat_id) do update 
-    set telegram_chat_id = $1`,
+    on conflict (telegram_chat_id) do nothing`,
```

Comment:
how come we're changing this?

Reply:
these are functionally equivalent but the latter doesn't require an extra db write